lib/v2/xiaomiyoupin/crowdfunding.js

```diff
@@ -0,0 +1,40 @@
+const got = require('@/utils/got');
+
+const base_url = 'https://m.xiaomiyoupin.com';
+module.exports = async (ctx) => {
+    const resp = await got('https://home.mi.com/lasagne/page/5');
+    const site_url = resp.data.redirect.location;
+
+    const urlParams = new URLSearchParams(site_url);
+    const pageid = urlParams.get('pageid');
+    const sign = urlParams.get('sign');
+
+    // 1. fetchPageData
+    const pageData = await got(`${base_url}/mtop/navi/venue/page?page_id=${pageid}&pdl=jianyu&sign=${sign}`);
+    const crowd_funding_floor = pageData.data.data.floors.find((floor) => floor.module_key === 'crowding');
+    const query_list = crowd_funding_floor.query_list;
+
+    // 2. fetchFloorDetailData
+    const floor_detail_data = await got.post(`${base_url}/mtop/navi/venue/batch?page_id=${pageid}&pdl=jianyu&sign=${sign}`, {
+        json: {
+            query_list,
+        },
+    });
```

Comment:
I think you can reuse the code in `latest.js` but a `module_key` filter of `crowd_funding`